samples/firefox-accounts/build.gradle

```diff
@@ -38,7 +38,7 @@ android {
 dependencies {
     implementation project(':service-firefox-accounts')
 
-    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
+    implementation Deps.support_constraintlayout
```

Comment:
nit: unrelated change. Let's always file a separate issue for that to not make the review diff too large.